pypdf/_utils.py

```diff
@@ -110,6 +110,31 @@ def parse_iso8824_date(text: Optional[str]) -> Optional[datetime]:
     raise ValueError(f"Can not convert date: {orgtext}")
 
 
+def format_datetime_to_pdf(dt: datetime) -> str:
```

Comment:
I am not sure about the name. Why aren't we matching this with the `parse_iso8824_date` name which does to parsing?